src/claudia.py

```diff
@@ -1732,7 +1732,7 @@ def slot_room_create(self):
 
     @pyqtSlot()
     def slot_room_delete_m(self):
-        room_name = self.sender().text()
+        room_name = self.sender().proeprty("data")
```

Comment:
@falkTX @net147 : typo ?

Reply:
Discard that, already fixed [in here](https://github.com/falkTX/Cadence/commit/4a5f5b445bfb3cd23a797c172dfbcb42e0cf114f)